packages/core/src/types/IViewport.ts

```diff
@@ -360,6 +363,18 @@ interface IViewport {
   getCurrentImageIdIndex(): number;
   /** gets the positional slice location in the view, similar to scrollbar, the top image is 0, the bottom is getNumberOfSlices - 1 */
   getSliceIndex(): number;
+
+  /**
+   * Gets the slice view information
+   */
+  getSliceViewInfo(): {
+    width: number;
+    height: number;
+    sliceIndex: number;
+    slicePlane: number;
```

Comment:
Could you document slicePlane please - it took me a moment to figure out what that was.  The rest seem self explanatory.